### ISSUE TITLE
empirical example

### ISSUE THREAD

**Issue by corybrunson** [julia-wrobel/registr#4]
The vignette includes sufficient examples to get a user started, but the package only demonstrates the registration functionality on simulated datasets, so i had trouble developing a sense of how robust/adaptable it is. I recognize that the application from [the paper](http://juliawrobel.com/Downloads/registration_ef.pdf) uses sensitive data, and i had trouble myself finding a suitable (multivariate binary time series) dataset to experiment with. (Ultimately i binarized the `influenza` dataset from the **tscount** package for some experiments.) Would it be feasible to include a suitable empirical dataset, and examples based on it?

_This issue is part of a [JOSS review](https://github.com/openjournals/joss-reviews/issues/557) of this package._

**Comment by julia-wrobel** [comment]
I added a data folder and in it included a subset of accelerometer data from the publicly available national health and nutrition examination survey (NHANES) data. This is the same style of data that we used in our methods paper and I processed it so that it works with our software. I also included an example using this dataset in the documentation for the register_fpca() function. Does that work?
